src/main/java/de/zalando/aruha/nakadi/controller/EventTypeController.java

```diff
@@ -192,6 +193,13 @@ private void validateSchema(final EventType eventType) throws InvalidEventTypeEx
                 throw new InvalidEventTypeException("\"metadata\" property is reserved");
             }
 
+            List<String> absentFields = eventType.getPartitionKeyFields().stream()
```

Comment:
We can't change schema or partition keys in update. So it's working only for update.